theme/keywind/login/assets/providers/discord.ftl

```diff
@@ -0,0 +1,35 @@
+<#-- https://discord.com/branding -->
+<#macro kw name="Discord">
+    <svg
+      version="1.1"
+      id="Layer_1"
+      xmlns="http://www.w3.org/2000/svg"
+      xmlns:xlink="http://www.w3.org/1999/xlink"
+      x="0px"
+      y="0px"
+      viewBox="0 0 150 150"
+      style="enable-background: new 0 0 150 150"
+      xml:space="preserve"
+    >
+      <style type="text/css">
+        .st0 {
+          fill: #5865f2;
+        }
+      </style>
+      <g id="图层_2">
+        <g id="Discord_Logos">
+          <g id="Discord_Logo_-_Large_-_White">
+            <path
+              class="st0"
+              d="M119.1,34.9c-8.4-3.8-17.2-6.5-26.2-8.1c-1.2,2.2-2.4,4.5-3.4,6.8c-9.6-1.5-19.5-1.5-29.1,0
+               c-1-2.3-2.1-4.6-3.4-6.8c-9.1,1.5-17.9,4.3-26.2,8.1C14.2,59.5,9.7,83.4,12,107l0,0c9.7,7.2,20.6,12.6,32.2,16.2
+               c2.6-3.5,4.9-7.2,6.9-11.1c-3.8-1.4-7.4-3.1-10.9-5.2c0.9-0.7,1.8-1.3,2.7-2c20.4,9.6,44,9.6,64.3,0c0.9,0.7,1.8,1.4,2.7,2
+               c-3.5,2-7.1,3.8-10.9,5.2c2,3.9,4.3,7.6,6.9,11.1c11.6-3.5,22.5-9,32.2-16.1l0,0C140.7,79.7,133.5,55.9,119.1,34.9z M53.9,92.5
+               c-6.3,0-11.5-5.7-11.5-12.7s5-12.7,11.4-12.7s11.6,5.7,11.5,12.7S60.3,92.5,53.9,92.5z M96.1,92.5c-6.3,0-11.4-5.7-11.4-12.7
+               s5-12.7,11.4-12.7s11.5,5.7,11.4,12.7S102.5,92.5,96.1,92.5z"
+            />
+          </g>
+        </g>
+      </g>
+    </svg>
```

Comment:
Hey @ATechAdventurer 👋

Thanks for your contribution. Please update the icon for consistency with others.

```suggestion
  <svg viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
    <title>${name}</title>
    <path d="M24 12C24 5.37258 18.6274 0 12 0C5.37258 0 0 5.37258 0 12C0 17.9895 4.38822 22.954 10.125 23.8542V15.4688H7.07813V12H10.125V9.35626C10.125 6.34875 11.9165 4.68751 14.6576 4.68751C15.9705 4.68751 17.3438 4.92188 17.3438 4.92188V7.875H15.8306C14.3399 7.875 13.875 8.80002 13.875 9.74901V12H17.2031L16.6711 15.4688H13.875V23.8542C19.6118 22.954 24 17.9895 24 12Z" fill="#1877F2" />
    <path d="M16.6711 15.4688L17.2031 12H13.875V9.74901C13.875 8.80002 14.3399 7.875 15.8306 7.875H17.3438V4.92188C17.3438 4.92188 15.9705 4.68751 14.6576 4.68751C11.9165 4.68751 10.125 6.34875 10.125 9.35626V12H7.07813V15.4688H10.125V23.8542C10.7453 23.9514 11.3722 24.0002 12 24C12.6379 24 13.2641 23.9501 13.875 23.8542V15.4688H16.6711Z" fill="white" />
  </svg>
```